ts/parts/PieSeries.ts

```diff
@@ -1051,29 +1051,37 @@ seriesType<Highcharts.PieSeries>(
         drawEmpty: function (this: Highcharts.PieSeries): void {
             var centerX,
                 centerY,
+                start = this.startAngleRad,
+                end = this.endAngleRad,
                 options = this.options;
 
             // Draw auxiliary graph if there're no visible points.
             if (this.total === 0) {
                 centerX = this.center[0];
                 centerY = this.center[1];
 
-                if (!this.graph) { // Auxiliary graph doesn't exist yet.
-                    this.graph = this.chart.renderer.circle(centerX,
-                        centerY, 0)
+                if (!this.graph) {
+                    this.graph = this.chart.renderer
+                        .arc(centerX, centerY, this.center[1] / 2, 0, start, end)
                         .addClass('highcharts-graph')
                         .add(this.group);
                 }
 
-                this.graph.animate({
+                this.graph.attr({
```

Comment:
Why animation is removed?

Reply:
Getting rid of the animation from this place resulted from my conversation with @Torstein - the purpose of the series animation is primarily to draw the user's attention and invite him to interact (there is nothing to pay attention to in the empty graphics).

Reply:
Getting rid of the animation from this place resulted from my conversation with @TorsteinHonsi - the purpose of the series animation is primarily to draw the user's attention and invite him to interact (there is nothing to pay attention to in the empty graphics).